Meta/build-image-qemu.sh

```diff
@@ -183,14 +183,14 @@ cleanup() {
                 fusermount -u mnt || (sleep 1 && sync && fusermount -u mnt)
             else
                 if [ "$(uname -s)" = "FreeBSD" ]; then
-                    ;
+                    echo umount mnt
```

Comment:
You should be able to invert the boolean check to avoid the need to do stuff like this to get past the linter?

Reply:
Also please squash these linter changes into their respective original commits.

Reply:
put in an update that hopefully the linter finds acceptable

Reply:
Brian's recommendation was more to do something like

```
if [ ! <uname = FreeBSD> ]; then
   actually umount
fi
```
rather than having two branches to the condition, one of which is a no-op.

Reply:
thank you for the clarification on the recommendation, it makes more sense to have branch logic on reflection. i have updated the logic per suggestion.